source/_docs/ecosystem/caddy.markdown

```diff
@@ -10,12 +10,28 @@ footer: true
 ---
 
 Configure [Caddy Server](https://caddyserver.com/) for use as a reverse proxy to Home Assistant.
-
+## Install
+For the one step installer type:
+```
+curl https://getcaddy.com | bash -s personal
+```
+Then create a file named ``Caddyfile``:
 ```
-hass.example.org {
```

Comment:
example.org makes more sense here in my opinion since it's a domain designed for examples. I'd be concerned users might mistake yourdomain.org as an actual domain.